src/lib/interfaces.ts

```diff
@@ -0,0 +1,7 @@
+export interface AlarmsMetrics {
```

Comment:
You should create a folder dedicated to `interfaces`. Inside that, you can create an `AlarmMetrics` file, just so they are separate

Reply:
I don't think this interface name is appropriate. A more appropriate name might be `TrackedMetric` or just `Metric` since this isn't exactly related to alarms, it is also part of the configuration. But actually, since they are referred to as `Parameter` in the specs in most docs I have seen, we can name it that for now.

Reply:
There is more complication in those themselves and I will work on creating some interfaces so you can name this whatever for now and use what I add later.